### PR TITLE
feat: Acl button access block

### PR DESCRIPTION
**Pull request by PritishBudhiraja**
## Type of Change

- [ ] Bugfix
- [x] New feature
- [ ] Enhancement
- [ ] Refactoring
- [ ] Dependency updates
- [ ] Documentation
- [ ] CI/CD

## Checklist

- [x] I ran `npm run re:build`
- [x] I reviewed submitted code
- [ ] I added unit tests for my changes where possible

### CODE REVIEW

File: src/screens/HyperSwitch/Connectors/ConnectorList.res [hunk 1]
```diff
@@ -146,35 +157,27 @@ module NewProcessorCards = {
...
-                toolTipFor={<div
-                  className="p-2 cursor-pointer" onClick={_ => handleClick(connectorName)}>
-                  <GatewayIcon gateway={connectorName->String.toUpperCase} className=size />
-                </div>}
-                toolTipPosition={Top}
-                tooltipWidthClass="w-30"
-              />
+              <div
+                className="p-2 cursor-pointer"
+                onClick={_ =>
```
Comment (JeevaRamu0104) [inline src/screens/HyperSwitch/Connectors/ConnectorList.res:171]: can we remove this check

File: src/screens/HyperSwitch/Refunds/ShowRefund.res [hunk 2]
```diff
@@ -118,17 +119,19 @@ let make = (~id) => {
...
-        actualData=orderData
-        entity={OrderEntity.orderEntity}
-        resultsPerPage=1
-        showSerialNumber=true
-        totalResults=1
-        offset
-        setOffset
-        currrentFetchCount=1
-      />
+      <UIUtils.RenderIf condition={userPermissionJson.paymentRead !== NoAccess}>
```
Comment (JeevaRamu0104) [inline src/screens/HyperSwitch/Refunds/ShowRefund.res:122]: Are we want completely not to show the table itself

Comment (PritishBudhiraja) [reply src/screens/HyperSwitch/Refunds/ShowRefund.res:122]: Yes Jeeva else the API Call will happen and then the that will be in error state

File: src/screens/HyperSwitch/Connectors/ConnectorList.res (hunk 1 above)
Comment (PritishBudhiraja) [reply src/screens/HyperSwitch/Connectors/ConnectorList.res]: updated

**Review by JeevaRamu0104 — APPROVED** [review]

**Review by JeevaRamu0104 — APPROVED** [review]

**Review by Riddhiagrawal001 — APPROVED** [review]

**Pull request merged**